webapp/src/components/App/Singlestudy/explore/Xpansion/Settings/SettingsForm.tsx

```diff
@@ -220,6 +220,30 @@ function SettingsForm(props: PropType) {
             sx={{ mb: 1 }}
           />
         </Box>
+        <Box
+          sx={{
+            display: "flex",
+            alignItems: "flex-start",
+            width: "100%",
+            mb: 2,
+            "&> div": {
+              mr: 2,
+              mb: 2,
+            },
+          }}
+        >
+          <TextField
+            type="number"
+            label={t("xpansion.separationParameter")}
+            variant="filled"
+            value={currentSettings.separation_parameter}
+            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
+              handleChange("separation_parameter", parseFloat(e.target.value))
+            }
+            sx={{ mb: 1 }}
+            InputProps={{ inputProps: { min: 0, max: 1 } }}
+          />
+        </Box>
```

Comment:
![image](https://github.com/AntaresSimulatorTeam/AntaREST/assets/33469289/4355882a-c97a-4c58-8d4c-765a01e19ed4)

The `separationParameter` Field is not consistent with the rest of the UI, consider moving it to the parent container and removing the `Box` that wrap it. Also when you will do this please not that `InputProps={{ inputProps: { min: 0, max: 1 } }}` will alter the Field styles, you will have to adjust the width most likely using a `minWidth` or any other solution that suits you.

Reply:
```javascript
     <Box
          sx={{
            display: "flex",
            alignItems: "flex-start",
            width: "100%",
            mb: 2,
            "&> div": {
              mr: 2,
              mb: 2,
              flexGrow: 1,
            },
          }}
        >
          <TextField
            type="number"
            label={t("xpansion.relativeGap")}
            variant="filled"
            value={currentSettings.relative_gap || ""}
            onChange={(e) =>
              handleChange("relative_gap", parseFloat(e.target.value))
            }
            sx={{ mb: 1, width: 1 }}
          />
          <SelectFields sx={{ mb: 1, width: 1 }}>
            <SelectSingle
              name="solver"
              list={solver.map((item) => {
                return { id: item, name: item };
              })}
              label={t("xpansion.solver")}
              data={currentSettings.solver || ""}
              handleChange={handleChange}
              optional
              sx={{
                minWidth: "100%",
              }}
            />
          </SelectFields>
          <TextField
            type="number"
            label={t("xpansion.batchSize")}
            variant="filled"
            value={currentSettings.batch_size || ""}
            onChange={(e) =>
              handleChange("batch_size", parseInt(e.target.value, 10))
            }
            sx={{ mb: 1, width: 1 }}
          />
          <TextField
            type="number"
            label={t("xpansion.timeLimit")}
            variant="filled"
            value={Math.round((currentSettings.timelimit || 1e12) / 3600)}
            onChange={(e) =>
              handleChange(
                "timelimit",
                Math.round(parseFloat(e.target.value) * 3600),
              )
            }
            sx={{ mb: 1, width: 1 }}
          />
          <TextField
            type="number"
            label={t("xpansion.logLevel")}
            variant="filled"
            value={currentSettings.log_level || ""}
            onChange={(e) =>
              handleChange("log_level", parseInt(e.target.value, 10))
            }
            sx={{ mb: 1, width: 1 }}
          />
          <TextField
            type="number"
            label={t("xpansion.separationParameter")}
            variant="filled"
            value={currentSettings.separation_parameter}
            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
              handleChange("separation_parameter", parseFloat(e.target.value))
            }
            sx={{ mb: 1, width: 1 }}
            inputProps={{ min: 0, max: 1 }}
          />
        </Box>
```

- move the Field to the parent container
- add `flexGrow: 1` to the parent
- ensure consistent styles adding `width: 1` to all childs
- use ` inputProps={{ min: 0, max: 1 }}` to ensure validation rules